doc/specs/Configuration object for profiles.md

```diff
@@ -0,0 +1,132 @@
+of further parameters can be discussed in the future and is out of scope for this spec.
+
+### Inheritance
+
+In the case that not all of the allowed parameters are defined in an appearance object, we will obtain the
+values for those parameters in the following matter:
+
+If the profile defines an `unfocusedConfig`, any parameters not explicitly defined within it will adopt
+the values from the profile itself. If the profile does not define an `unfocusedConfig`, then the global/default `unfocusedConfig` is used
+for this profile.
+
+Thus, if a user wishes for the unfocused state to look the same as the focused state for a particular profile,
+while still having a global/default unfocused state appearance, they simply need to define an empty `unfocusedConfig`
+for that profile (similarly, they could define just 1 or 2 parameters if they wish for minimal changes between the focused
+and unfocused states for that profile). If they do not define any `unfocusedConfig` for the profile, then
+the global/default one will be used.
+
+If neither the profile nor the global settings have defined an `unfocusedConfig`, then there will be no
+appearance change when switching between focused and unfocused states.
+
+## UI/UX Design
+
+Users will be able to add a new setting to their profiles that will look like this:
+
+```
+"unfocusedConfig":
+{
+    "colorScheme": "Campbell",
+    "cursorColor": "#888",
+    "cursorShape": "emptyBox",
+    "foreground": "#C0C0C0",
+    "background": "#000000"
+}
+```
+
+## Capabilities
+
+### Accessibility
+
+Does not affect accessibility.
+
+### Security
+
+Does not affect security.
+
+### Reliability
+
+This is another location in the settings where parsing/loading the settings may fail. However, this is the case
+for any new setting we add so I would say that this is a reasonable cost for this feature.
+
+### Compatibility
+
+Should not affect compatibility.
+
+### Performance, Power, and Efficiency
+
+## Potential Issues
+
+Inactive tabs will be 'rendered' in the background with the `UnfocusedRenderingParams` object, we need to make
+sure that switching to an inactive tab (and so causing the renderer to update with the 'normal' parameters)
+does not cause the window to flash/show a jarring indicator that the rendering values changed.
+
+When certain appearance settings are changed via OSC sequences (such as the background color), only the focused/regular
+appearance will change and the unfocused one will remain unchanged. This means that even if no unfocused configuration was set
+(i.e. the user simply wants the unfocused configuration to look like the regular appearance), there will end up being
+a difference between both appearances.
+
+## Future considerations
+
+We will need to decide how this will look in the settings UI.
+
+We may wish to add more states in the future (like 'elevated'). When that happens, we will need to deal with how

```

Comment:
Should we add some notes on why this was such a pain of a design, and why it _wasn't_ included now?